packages/api-client/src/main/conversation/Conversation.ts

```diff
@@ -48,6 +48,6 @@ export interface Conversation {
   members: ConversationMembers;
   message_timer: number | null;
   name: string;
-  team?: string;
+  team?: string | null;
```

Comment:
optional covers `null` already

Reply:
In general I don't like a distinction between `null` and `undefined` here because there should be none. This statement can lead to unexpected interpretations of null/undefined.

Reply:
Why to you think it can led to unexpected interpretations? The typings now reflect 100% the behaviour of the backend because the backend sends this property with a value of `null`. It never sends `undefined` or leaves out the property completely.

Reply:
The backend doesn't really care about the differences of undefined/null what they want to express is that there is no value. So we should do the same.

Reply:
What we should do is creating an accurate typing of the backend payload. Does the backend return `undefined` for the "team" property? No. Does the backend return `null` for the "team" property? Yes. So creating a typing which allows `undefined` is just wrong.